be2-scala/docs/README.md

```diff
@@ -85,7 +85,7 @@ The results are then collected by the main merger (blue "merger" circle) and sen
 ### Sending and Receiving Messages
 
 `ClientActor` is complex yet wonderful piece of code that resembles black magic at first sight. It *is* conceptually both the (websocket) link between the server & a particular client or server, as well as the actual internal representation of the client or server. Each different client or server is represented by a unique `ClientActor`. It serves as both the entry point (receiving a `JsonRpcRequest`) and exit point (sending a `JsonRpcResponse` to a client or broadcasting a `Broadcast` to multiple clients) of the graph.
-
+Whenever a client disconnects from the server, the clienActor associated to that connection gets killed and a new clientActor is recreated from scratch
```

Comment:
Would that not be better to use `connection is removed` or `connection is deleted` instead of `killed` ?